### ISSUE TITLE
Iterating over function schedules raises error

### ISSUE THREAD

**Issue by MortGron** [cognitedata/cognite-sdk-python#1972]
**System information (please complete the following information):**
 - OS: Windows 11 Enterprise 23H2
 - Python Version: CPython/3.12.4
 - SDK Version: 7.62.7

**Describe the bug**
```
with client.function.schedules():
    pass
```
fails with

`CogniteAPIError: {'cursor': ['Unknown field.']} | code: 400`

Does the API support pagination for function schedules?

**Comment by MortGron** [comment]
@HMEiding does the schedules list endpoint support pagination?

**Comment by HMEiding** [comment]
I'm assuming you mean
```
for schedule in client.functions.schedules():
    pass
```

Yes, you're right, the `.../schedules/list` endpoint does not support pagination. However, this can be fixed on the SDK level, as was done for `client.functions()`.

**Comment by HMEiding** [comment]
Possible fix in #2016 .

**Comment by HMEiding** [comment]
Fixed in #2016
